cmd/snap/cmd_services.go

```diff
@@ -53,6 +56,14 @@ var (
 	longServicesHelp  = i18n.G(`
 The services command lists information about the services specified, or about
 the services in all currently installed snaps.
+
+If executed as root user, the 'Startup' column of any user service will be whether
+it's globally enabled (i.e systemctl is-enabled). To view the actual 'Startup'|'Current'
+status of the user services for the root user itself, --user can be provided.
+
+If executed as a non-root user, the 'Startup'|'Current' status of user services 
+will be the current status for the invoking user. To view the global enablement
+status of user services, --global can be provided.
```

Comment:
@degville Could you take a look at this? I need to see if this text makes any sense. There are two new switches introduced for `snap services` that describes what the `Startup` and `Current` fields mean